di-ipv-credential-issuer-stub/src/main/resources/templates/authorize.mustache

```diff
@@ -66,6 +72,8 @@
             </details>
         {{/shared_claims}}
 
+        <div class="govuk-form-group" id="test_data_block"></div>
```

Comment:
There's a possibility this will break the smoke tests as it may alter the x-paths to the input fields below. It depends how the smoke tests are finding the elements.
It might be worth running this up locally and checking the xpaths the smoke tests are using still find the right things.

Reply:
Although looking at the tests, they're using id's rather than xpaths on that page so it might be ok...
https://github.com/alphagov/di-ipv-core-tests/blob/main/src/test/java/gov/di_ipv_core/pages/SupplyDataPage.java